nifi-nar-bundles/nifi-gcp-bundle/nifi-gcp-processors/src/main/java/org/apache/nifi/processors/gcp/bigquery/PutBigQueryBatch.java

```diff
@@ -257,17 +257,19 @@ public void onTrigger(ProcessContext context, ProcessSession session) throws Pro
         }
 
         final String projectId = context.getProperty(PROJECT_ID).evaluateAttributeExpressions().getValue();
+        final String designateProjectId = context.getProperty(DESIGNATE_PROJECT_ID).evaluateAttributeExpressions().getValue();
```

Comment:
```suggestion
        final String designateProjectId = context.getProperty(DESIGNATE_PROJECT_ID).evaluateAttributeExpressions(flowFile).getValue();
```

I believe this is required since you defined FlowFile Attributes scope in the property definition.